plugins/pulp_docker/plugins/distributors/distributor_export.py

```diff
@@ -92,9 +92,12 @@ def validate_config(self, repo, config, config_conduit):
         :return: tuple of (bool, str) to describe the result
         :rtype:  tuple
         """
+        repo = model.Repository.objects.get(repo_id=repo_transfer.id)
+        repo.repo_id = repo_transfer.id
```

Comment:
yeah, unfortunately this pattern is required until @asmacdo's updates are merged that rename the repo.id to repo.repo_id